mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulFollowingScreen.kt

```diff
@@ -95,7 +97,7 @@ private class SnapshotSocialScreenState(
     override var mode: SocialScreenState.Mode,
     override val searchFieldInteraction: MutableInteractionSource,
     private val scope: CoroutineScope,
-    private val onPersonClick: (Person) -> Unit
+    private val onPersonElementClick: (Person) -> Unit
```

Comment:
You should make this a `(ProfileAdapter) -> Unit`, so you don't have to add `uid` to `Person`.